packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -121,6 +122,12 @@ class CupertinoPicker extends StatefulWidget {
   ///
   /// This can be set to null to disable the background painting entirely; this
   /// is mildly more efficient than using [Colors.transparent].
+  ///
+  /// The [backgroundColor] is used to create the gradient of the picker. The
```

Comment:
'the gradient' might not be a familiar concept to readers. Point out that it's an edge fade effect gradient.